metabase/metabase
# Fix sunburst color assignment that causes pie chart crashing

Closes https://github.com/metabase/metabase/issues/49160

### Description

Pie charts crash on instances where palette has overriden light or dark accents. Caused by [this](https://github.com/metabase/metabase/pull/49183/files#diff-e0af4fe407c3b55c81c1736ac64a3a1fd8e13d24b946b8bae3b27ab2eecf699dL15) line which assumes the global `color` object includes only base accents like `accent7` and not `accent7-dark`. 

Also fixes shoring an error instead of a pie placeholder.

### How to verify

Colors:
- Run Metabase EE
- Go to admin settings -> appearance
- Change light/dark accents
- Create any pie chart -> Ensure it does not crash

Placeholder:
- New question -> Orders
- Without grouping change display to Pie
- Ensure it shows a pie chart placeholder instead of an error

### Checklist
